SokolTextGame/World.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Input;
+
+namespace SokolTextGame
+{
+    internal class World
```

Comment:
To resolve the private/public issue, you need to make World public, not internal.